io.openems.backend.common/src/io/openems/backend/common/debugcycle/DebugLoggable.java

```diff
@@ -0,0 +1,42 @@
+package io.openems.backend.common.debugcycle;
+
+import static java.util.Collections.emptyMap;
+
+import java.util.Map;
+
+import com.google.gson.JsonElement;
+
+import io.openems.backend.common.timedata.Timedata;
+
+public interface DebugLoggable {
+
+	// TODO should be merged with OpenemsComponent
+	/**
+	 * Returns a unique ID for this OpenEMS component.
+	 *
+	 * @return the unique ID
+	 */
+	public String id();
+
+	/**
+	 * Gets some output that is suitable for a continuous Debug log. Returns 'null'
+	 * by default which causes no output.
+	 *
+	 * @return the debug log output
+	 */
+	public default String debugLog() {
```

Comment:
If a Component doesn't want to have a debugLog it shouldn't implement the interface
-> no need for a default implementation